### PR TITLE
zh-Hant: Create `po/zh-Hant.po`

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #684.

### CODE REVIEW

File: po/zh-Hant.po [hunk 1]
```diff
@@ -0,0 +1,18441 @@
...
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-05-26 15:20+0200\n"
+"Last-Translator: Martin Geisler <mgeisler@google.com>\n"
+"Language-Team: Chinese (traditional) <zh-l10n@lists.linux.org.tw>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: zh-Hans\n"
```
Comment (hueich) [inline po/zh-Hant.po:11]: Should be zh-Hant

Comment (mgeisler) [reply po/zh-Hant.po:11]: Oh my, I'm sorry... I hand-edited this and mixed it up!

```suggestion
"Language: zh-Hant\n"
```

**Review by hueich — APPROVED** [review]
Approved once comment is resolved.

File: po/zh-Hant.po [hunk 2]
```diff
@@ -0,0 +1,18441 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-05-26 15:20+0200\n"
+"Last-Translator: Martin Geisler <mgeisler@google.com>\n"
+"Language-Team: Chinese (traditional) <zh-l10n@lists.linux.org.tw>\n"
```
Comment (mgeisler) [inline po/zh-Hant.po:7]:
```suggestion
"Language-Team: Chinese (Traditional) <zh-l10n@lists.linux.org.tw>\n"
```

**Pull request merged**